scripts/new_project.kts

```diff
@@ -9,23 +9,67 @@ object NewProject {
     private const val TEMPLATE_FOLDER_NAME = "CoroutineTemplate"
     private const val TEMPLATE_PACKAGE_NAME = "co.nimblehq.coroutine"
 
+    private const val APP_PATTERN = "^[A-Z\$][a-zA-Z0-9(\\s)\$]*\$"
+    private const val PACKAGE_PATTERN = "^[a-z]+(\\.[a-z][a-z0-9]*)*\$"
+
     private val modules = listOf("app", "data", "domain")
 
     private var appName = ""
-    private val appNameWithoutSpace: String
-        get() = appName.replace(" ", "")
```

Comment:
Could we please rebase? I don't think we can remove this, as we require it for renaming the application class 🙏